spec/unit/mutant/ast/named_children_spec.rb

```diff
@@ -42,8 +42,18 @@
       end
 
       describe '#bar' do
-        it 'returns named child bar' do
-          expect(instance.bar).to be(node_bar)
+        context 'when node is present' do
+          it 'returns named child bar' do
+            expect(instance.bar).to be(node_bar)
+          end
+        end
+
+        context 'when node is NOT present' do
```

Comment:
In general I don't think it's necessary to emphasize `NOT` in this way but I don't feel that strongly about it.

I do with there was a convention to emphasize things like this in a spec description. Perhaps wrapping the word in asterisks?